From b0d2d6650bb4924f4dfc3b429835b9ba19f1067e Mon Sep 17 00:00:00 2001
From: Mark Asselstine <mark.asselstine@windriver.com>
Date: Wed, 14 May 2014 09:29:35 -0400
Subject: [PATCH] gemspec: don't force the use of gem-private_key.pem

This will have security implications but it doesn't affect
core functionality and allows the build to complete when
this certificate is not available.

Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
---
 net-ssh.gemspec | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/net-ssh.gemspec b/net-ssh.gemspec
index edbbeaa..856bc1e 100644
--- a/net-ssh.gemspec
+++ b/net-ssh.gemspec
@@ -180,7 +180,9 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.rubyforge_project = "net-ssh"
   s.rubygems_version = "1.8.23"
-  s.signing_key = "/mnt/gem/gem-private_key.pem"
+  if File.exists?("/mnt/gem/gem-private_key.pem")
+    s.signing_key = "/mnt/gem/gem-private_key.pem"
+  end
   s.summary = "Net::SSH: a pure-Ruby implementation of the SSH2 client protocol."
 
   if s.respond_to? :specification_version then
-- 
1.8.3.2

